### DISCUSSION TITLE
Where to place downloaded files

### DISCUSSION THREAD

**Discussion by hesselmans** [dtcenter/METplus#1937] [Installation] [answered]
I have downloaded tar_files.tar and met-10.1.2.20220516.tar.gz.
Should both files be placed in the same folder?

Following the instructions I can run first part of compile_MET_all.sh.
First part includes compilation of libraries in tar_files.tar.
Then it stops with:
./compile_MET_all.sh: line 888: ./configure: No such file or directory

My guess is I placed files in the wrong folder.
Hence my question above.
Pls can you elaborate on where files should be when starting compile_MET_all.sh

**Comment by jprestop** [comment, marked as answer]
HI @hesselmans.

Thank you for your question.

> I have downloaded tar_files.tar and met-10.1.2.20220516.tar.gz.
Should both files be placed in the same folder?

The tar_files.tar file should be unpacked:
```
tar -xvf tar_files.tar
```
and the met-10.1.2.20220516.tar.gz should be placed in the tar_files directory.  The tar_files directory should be on the same level as the compile_MET_all.sh script, along with the configuration file for compile_MET_all.sh. I hope that helps.

Please let us know where/how we can make our documentation more clear so that it is less confusing for users.  I am not sure where you are looking at instructions, as we have them in a couple of places:

- The [MET Download page](https://dtcenter.org/community-code/model-evaluation-tools-met/download)
- Our [Compiling MET training page](https://metplus-training.readthedocs.io/en/latest/modules/Environment/met_installation.html)

Please let us know if you have any further questions.
